### DISCUSSION TITLE
Default branch is now `main`

### DISCUSSION THREAD

**Discussion by chalin** [google/docsy#913] [General]
We've renamed the default branch from `master` to `main` via #765. If you have a PR pending, GitHub will handle the rename automatically. GitHub warns that it ...

> ### Will not update ... members' local environments
> ... members will have to manually update their local environments. We'll let them know when they visit the repository, or you can share the following commands.
> ```sh
> git branch -m master main
> git fetch origin
> git branch -u origin/main main
> git remote set-head origin -a
> ```

Follow the steps above to handle the rename of the default branch for your local repository.

**Comment by chalin** [comment]
Note that if you have a fork, visit your fork repo on GitHub and rename the repo's default branch to `main` first. Then you can follow the instructions given above to rename `master` to `main` in a local clone.

**Comment by doganulus** [comment]
This change may break `module` setup?

By default `v0.1.0` had been downloaded when I tried to set up using `hugo` modules. Then I have needed to run `hugo mod get github.com/google/docsy@main` to get the latest version.

**Reply by chalin** [reply]
Thanks for pointing that out. Note that while official support for Hugo modules is on its way, it wasn't meant to be supported for v0.1.0 (but that is probably beside the point that you were trying to make).
